Core/src/com/serotonin/m2m2/db/dao/PointValueDaoSQL.java

```diff
@@ -141,6 +157,9 @@ public PointValueTime savePointValueSync(DataPointVO vo, PointValueTime pointVal
     @Override
     public void savePointValueAsync(DataPointVO vo, PointValueTime pointValue, SetPointSource source) {
         savePointValueImpl(vo, pointValue, source, true);
+        asyncCallsCounter.hit();
+        ASYNC_INSERTS_SPEED_COUNTER.setValue(asyncCallsCounter.getEventCounts()[0] / 1);
```

Comment:
When changing the bucket size don't forget to divide it down to save/s (so divide by 5)